unlock-js/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changes
 
+## Next Version (minor!)
+
+- createLock now returns a Promise of the deployed lock address
```

Comment:
Hum. Are these changes going to be included in this new module? They are breaking changes :/
You probably want to fork before all of these so you can publish your version